### DISCUSSION TITLE
Get users details by list of IDs

### DISCUSSION THREAD

**Discussion by AlTosterino** [zitadel/zitadel#6335] [Q&A] [answered]
Hey!

Is there possibility to hit `POST /users/_search` search API to search by list of IDs? 
Or is there any other endpoint that allows it? I saw that I can query above information within `List Organization Members` API endpoint, but it only works for users with permission to administrate ZITADEL on different levels. 

My current use case looks like this: For given list of user ids, get user details (names, emails etc) in 1 API fetch.

Thanks in advance.

**Comment by mffap** [comment, marked as answer]
I think this PR might be related to your question: https://github.com/zitadel/zitadel/pull/6210

**Reply by AlTosterino** [reply]
Thank you so much, I'll close this discussion then. I'm very happy that this is in progress <3
